lib/convert.js

```diff
@@ -167,7 +169,11 @@
         }
 
         // Write an image of format <format> to STDOUT
-        args.push(format + ":-");
+        if (format === "jpg" && useFlite && settings.useOptimalJPEG) {
```

Comment:
Yes, I'm suggesting that we might want all three (four?) options.  For testing/comparing if nothing else.